docs/topics/mta-web-config-jira-credentials.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-credentials_{context}"]
+= Configuring Jira credentials
+
+You need to configure a Jira credential for establishing a connection to Jira and creating Jira issues from within the {ProductName} ({ProductShortName}) {WebName} for tracking application migrations.
```

Comment:
A Tackle credential must defined to authenticate to a Jira instance's API.
Two types of credential are available, Basic auth (Jira Cloud or Jira Server/Datacenter) and Bearer Token (Jira Server/Datacenter).

Once a credential compatible with Jira has been created, it can be used to define a Jira instance in Tackle and establish connection to that instance.

Reply:
@gildub Thank you!
'Tackle' is an upstream name for MTA, isn't it? Can we call this credential 'MTA credential' instead?